### ISSUE TITLE
Incorrect NTLM message in Authorization header

### ISSUE THREAD

**Issue by mbeijen** [Legrandin/PyAuthenNTLM2#5]
I configured the latest version from Git HEAD on Ubuntu 10.04 LTS using included Apache, Python, and mod_python versions and got this error message when using IE7 on WinXP:

Incorrect NTLM message in Authorization header from address 172.24.30.115 and URI /foo:

What can this be?

**Comment by Legrandin** [comment]
Does that happen over an SSL connection?

**Comment by mbeijen** [comment]
No, this setup does not use SSL. Is this a requirement? Obviously I am aware of the security implications.

**Comment by Legrandin** [comment]
No, SSL is not required. I asked  because IE+SSL+NTLM do not work with the stock Apache configuration files, but I understand this is not your case.

Could you try with the latest code (I somewhat rearranged it)? Additionally, do you have KeepAlive set in Apache?

**Comment by zachyunl** [comment]
I have the same problem on FreeBSD 13.0? What can I do?
